README.md

```diff
@@ -158,7 +158,7 @@ defaults:
   cassandra:
     repository: "k8ssandra/cass-management-api"
   dse:
-    repository: "datastax/dse-server"
+    repository: "datastax/dse-mgmtapi-6_8"
```

Comment:
This requires upgrade test to verify that existing clusters do not break, lets say 6.8.9 installations.